packages/cmf/src/sagaRouter/documentTitle.js

```diff
@@ -0,0 +1,56 @@
+import { take } from 'redux-saga/effects';
+
+import matchPath from './matchPath';
+
+export function formatPath(path, parentPath) {
+	let fPath = '';
+	if (parentPath && parentPath === '/') {
+		fPath = `${parentPath}${path}`;
+	} else if (parentPath) {
+		fPath = `${parentPath}/${path}`;
+	} else {
+		fPath = path;
+	}
+	return fPath.replace(/[(]/g, '').replace(/[)]/g, '?');
```

Comment:
Could you add a comment about why ?
( react router v2 currently using but matchpatch of router v4 )